packages/@react-spectrum/s2/style/spectrum-theme.ts

```diff
@@ -671,7 +671,9 @@ export const style = createTheme({
     strokeWidth: {
       0: '0',
       1: '1',
-      2: '2'
+      2: '2',
+      3: '3',
```

Comment:
🤔 borderWidth only has 1, 2, and 4. 3 is a bit odd...

Reply:
true, i could do directly with custom value